src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -95,7 +95,7 @@ public async Task SynchronizeWithServer()
                 }
 
                 var projectKey = bindingConfiguration.Project.ProjectKey;
-                var serverBranch = await serverBranchProvider.GetServerBranchNameAsync();
+                var serverBranch = await serverBranchProvider.GetServerBranchNameAsync(CancellationToken.None);
```

Comment:
The taint synchronizer doesn't currently have a `CancellationTokenSource` so we can't pass anything except `CancellationToken.None` here.

I've added #3442 to cover adding cancellability.